pkg/cmd/upgrade/upgrade_boot.go

```diff
@@ -161,10 +161,14 @@ func (o *UpgradeBootOptions) Run() error {
 }
 
 func (o UpgradeBootOptions) createCommitForRequirements(requirementsFileName string) error {
-	err := o.Git().AddCommitFiles(o.Dir, "Merge jx-requirements.yml", []string{requirementsFileName})
+	err := o.Git().AddCommitFiles(o.Dir, fmt.Sprintf("Merge %s", requirementsFileName), []string{requirementsFileName})
 	if err != nil {
-		return errors.Wrapf(err, "error creating a commit with the merged jx-requirements.yml file from dir %s",
-			requirementsFileName)
+		if strings.Contains(err.Error(), "nothing to commit") {
```

Comment:
I don't think this is a good idea. We should not depend on checking the error message. I don't think the error message is part of the official git "API" and it also is dependent on the user's locale setting. We only had this case recently where code would fail on a computer with Swedish locale. ATM, we are enforcing `LC_ALL=ALL` when executing `git` from the command line. This works as a workaround, but the aim is to get away from matching error messages. 

It should be possible to do this by for example checking first whether there is something to commit.